src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -18,6 +19,7 @@ import { type ConnectedWallet } from '@/hooks/wallets/useOnboard'
 import { useCurrentChain } from '@/hooks/useChains'
 import { getTxOptions } from '@/utils/transactions'
 import { TxSimulation } from '@/components/tx/TxSimulation'
+import { txDispatch, TxEvent } from '@/services/tx/txEvents'
```

Comment:
As you can see, this component wasn't previously dispatching any tx events directly.
`txSender` does it. Let's create a function in txSender for this. Something like `dispatchSafeAppsTx`.

Reply:
ok done